src/h3/api/_api_template.py

```diff
@@ -61,9 +67,21 @@ def num_hexagons(resolution):
         return _cy.num_hexagons(resolution)
 
     def hex_area(resolution, unit='km^2'):
+        """ Return the average area of an H3 cell
+        for the given resolution.
+
+        todo: all cells, or just hexagons?
+        todo: `mean_hex_area`?
+        """
         return _cy.mean_hex_area(resolution, unit)
 
     def edge_length(resolution, unit='km'):
+        """ Return the average cell edge length
+        for the given resolution.
+
+        todo: all cells, or just hexagons?
```

Comment:
Presumably also for only hexagonal cells

Reply:
updating!